mozilla-release/mobile/android/base/java/org/mozilla/gecko/activitystream/homepanel/topnews/TopNewsAdapter.java

```diff
@@ -37,6 +37,8 @@
     private int numShowedNews = 0;
     private boolean isNewsExpanded;
 
+    private boolean isLightTheme;
```

Comment:
`mIsLightTheme`. Are we planning to add a boolean for every other theme we implement or are we using numeric coding?